test/commands/render_test.rb

```diff
@@ -123,7 +123,7 @@ def data
               'description' => 'A string with a regex pattern applied to it.',
               'type' => 'string',
               'example' => 'second',
-              'pattern' => '(^first$|^second$)'
```

Comment:
I think the pattern here should still have a pipe instead of the pre-escaped `&#x7c;`? Otherwise, we might no longer be testing the pipe escape?

Reply:
Oops. Good catch, thanks!